examples/flutter_gallery/lib/demo/animation/home.dart

```diff
@@ -129,9 +133,9 @@ class _SliverAppBarDelegate extends SliverPersistentHeaderDelegate {
 
   @override
   bool shouldRebuild(_SliverAppBarDelegate oldDelegate) {
-    return maxHeight != oldDelegate.maxHeight
-        || minHeight != oldDelegate.minHeight
-        || child != oldDelegate.child;
+    return maxHeight != oldDelegate.maxHeight ||
+        minHeight != oldDelegate.minHeight ||
+        child != oldDelegate.child;
```

Comment:
if we're working on the formatter then this is one area i think we could massively improve. `return` statements look way better in the adam-style. The style is: if you have a sequence of expressions at the same level each separated from the next by an operator (be it `&&`, `||`, `+`, `-`, or something else) then put the operator at the start of subsequent lines such that the expressions all end up vertically aligned.

Reply:
this would be for all cases, not just return statements correct? I'm willing to test it out as long as it is a simple switch.